app/app.py

```diff
@@ -8,10 +8,7 @@
 import uuid
 
 
-dynamodb = boto3.resource(
-    'dynamodb',
-    endpoint_url=os.environ['DYNAMODB_ENDPOINT_URL'],
```

Comment:
We cannot remove this line, need to specify it to run locally.
I'm gonna make it back later.

Reply:
Oh I see. I totally forgot about testing!